### ISSUE TITLE
Difficult to know where a message has been sent via Faceboook

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2552]
**Reported by ManuelRamos on 2012-07-23 14:14**
Using NVDA 2012.2.1 or latest snapshot with Faceboook and Mosilla Firefox. When sending a message, for example inviting anyone to be my facebook's friend, I can type a new message. However, when copying a link to the message or writing a not short text, we need to send it with space or enter keys.

It is happening that, when sending it, sometimes NVDA reports that message has been sent. But, in most situations, it appears just a message saying "alert" and we cannot know if the message was sent or not. Perheaps not, because cursor is positioned in place to edit the message.

After this initial try, it is not possible to get access back to send or cancel buttons.

**Comment by dkager** [comment]
While I'm not much of a Facebooker, this issue doesn't sound familiar to me with recent versions of the website in Firefox 52+. Nor did this ticket get much response. Closing.
